src/client/components/layouts/InstantSearch/ListeDesResultats.tsx

```diff
@@ -33,7 +33,7 @@ export const ListeDesResultats = React.forwardRef<HTMLElement | null, ListeDesRe
 						{resultats}
 						{pagination}
 						<Footnote htmlFor="partenaires-reference" id="partenaires" className={styles.footnote}>
-							les annonces listées ci-dessus nous sont fournies par nos partenaires (<a href="/cgu#3-services">liste disponible dans les <abbr title="Conditions Générales d'Utilisation">CGU</abbr></a>)
+							les annonces listées ci-dessus nous sont fournies par nos partenaires (<a href="/cgu#3.-services">liste disponible dans les <abbr title="Conditions Générales d'Utilisation">CGU</abbr></a>)
```

Comment:
Volontaire le point rajouté? 
C'est que Marked l'ignorait pour calculer l'id mais pas Markdown-it ?

Reply:
C'est ça !